<!DOCTYPE html>
<html><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
  <meta charset="utf-8">
  <title>Corina Duenas - CV</title>
  <link href="https://github.com/corinabioinformatic/corinabioinformatic.github.io/blob/master/mycv/epaCV_files/cv-screen.css" type="text/css" rel="stylesheet" media="screen">
  <link href="https://github.com/corinabioinformatic/corinabioinformatic.github.io/blob/master/mycv/epaCV_files/cv-print.css" type="text/css" rel="stylesheet" media="print">
<style>@media print {#ghostery-purple-box {display:none !important}}</style></head>
<body>

<!-- <div id="ghbutton">
<iframe src="http://ghbtns.com/github-btn.html?user=elipapa&repo=markdown-cv&type=fork"
  allowtransparency="true" frameborder="0" scrolling="0" width="53" height="20"></iframe>
</div> -->
  
  <div id="main">
    <div id="content">
    <h1 id="corina-duenas">Corina Duenas</h1>
<p>Computational biologist focused in data analysis. Working on Python and R modules applied to new Targets 
Discovery focused in autoimmune diseases. Experience in data-mining and statisics between different open databases. Background in pharmacy, immunology and bioinformatics.</p>

<div id="webaddress">
<script type="text/javascript">
//<![CDATA[
<!--
var x="function f(x){var i,o=\"\",l=x.length;for(i=0;i<l;i+=2) {if(i+1<l)o+=" +
"x.charAt(i+1);try{o+=x.charAt(i);}catch(e){}}return o;}f(\"ufcnitnof x({)av" +
" r,i=o\\\"\\\"o,=l.xelgnhtl,o=;lhwli(e.xhcraoCedtAl(1/)3=!08{)rt{y+xx=l;=+;" +
"lc}tahce({)}}of(r=i-l;1>i0=i;--{)+ox=c.ahAr(t)i};erutnro s.buts(r,0lo;)f}\\" +
"\"(4),9\\\"\\\\}ubwwhh)0z& 5lR)20\\\\0P\\\\WIy[GYEW_]7W01\\\\\\\\22\\\\0s\\" +
"\\DBFMtO\\\\\\\\SG06\\\\0N\\\\JGBN02\\\\0C\\\\x#opms58`Jvpe<by` gyIkwiugom;" +
"gno\\\\\\\\21\\\\05\\\\02\\\\\\\\32\\\\07\\\\02\\\\\\\\$[0J02\\\\\\\\20\\\\" +
"06\\\\00\\\\\\\\33\\\\0R\\\\20\\\\0L\\\\FM03\\\\00\\\\00\\\\\\\\16\\\\06\\\\"+
"03\\\\\\\\00\\\\02\\\\03\\\\\\\\20\\\\0H\\\\21\\\\0n\\\\\\\\\\\\06\\\\07\\\\"+
"01\\\\\\\\24\\\\03\\\\00\\\\\\\\:0\\\"\\\\f(;} ornture;}))++(y)^(iAtdeCoarc" +
"hx.e(odrChamCro.fngriSt+=;o27=1y%2;*=)yy)4+(9i>f({i+)i+l;i<0;i=r(foh;gten.l" +
"=x,l\\\"\\\\\\\"\\\\o=i,r va){,y(x fontincfu)\\\"\")"                        ;
while(x=eval(x));
//-->
//]]> </script>

| <a href="http://corinabioinformatic.github.io/">corinabioinformatic.github.io</a>
| <a href="http://uk.linkedin.com/in/corina-duenas-pharmaconetworks">LinkedIn</a>
| <a href="http://github.com/corinabioinformatic">GitHub</a>
</div>

<!-- notable accomplishments - ie. projects completed

link.opentargets.io
library-beam
www.targetvalidation.org
pnas paper
microbiome analysis for finch/seres
 -->

<h3 id="experienced-in">Experienced in</h3>

<p>Data analysis with python and R on biomedical data.
	Parsing JSON and XML files from biomedical open databases.</p>

<p>Collaboration in scientific projects (as a computational biologist), manual and automatized curation of openSource databases.</p>

<p><a href="https://reactome.org">Biomolecular networks</a></p>

<p>Analysis of GPA data - Granulomatosis with Polyangiitis project</p>

<p><a href="https://www.targetvalidation.org/">Drug target identification and prioritization for an orphan disease, mining and weightening data from different sources.</a> - Genetics, integrative and network-based approaches</p>

<!--<p>W</p>

<p>Machine learning for biomarker identification - human microbiome and immune cells subpopulations</p>

<p>Medicine - anesthesia, genetics, general medicine</p>-->

<h2 id="professional-experience">Professional Experience</h2>

<p><code class="highlighter-rouge">Sept2016 - Dic2017</code>
<strong>Data curator and data analyst<a href="https://reactome.org">Reactome</a></strong></p>

<p><code class="highlighter-rouge">March2016 - Sept2016</code>
<strong>Data curator &amp; Biomolecular Quantitative Networks <a href="https://www.ebi.ac.uk/biomodels-main/">Biomodels</a></strong></p>

<p>Projects:
1) Biomarkers discovery for autoimmune disease &amp; Granulomatosis with polyangitiis</p>

<p>My duties included curation of some mathematical models and writing monthly abstracts about determined models. Simultaneously I worked in a research project of an autoimmune disease (literature mining, reanalysis of current data) with perspectives to merge existing Biological Quantitative models to help to understand the disease.  Managing Databases as Reactome, OpenTargets, ArrayExpress, AtlasExpression, CHEMBL, ENSEMBL. I worked in a Linux environment with Python and R to manage big amounts of data.
Development follows an iterative and agile process, done in 
collaboration with scientist at all levels (science and engineers, and 
which requires a deep understanding of the immunology as well as the 
sources and tools available. My broad background (pharmacy, immunology and bioinformatics) helps me to bridge these diverse skill sets and communicate across disciplines.</p>

<p>2) <a href="https://reactome.org/">Reactome</a> </p>

<p>Working on computational biology and molecular networks analysis which leverage
 large scale human data sets with the goal of curate and update biological molecular network based on scientific literture. Special focus on interleukin pathways.</p>

<p>3) Development of the <a href="https://pypi.org/project/reactome-stats/">rectome_stats.py module</a></p>

<p>I am one of the Reactome metrics python developer, responsible to plan and manage the metrics of Reactome data and compare it with several other open resources to fill gaps and increase the level of detail in Reactome database by mean of other sources.</p>

<p>4) Master's degree project in pharmacodynamic evaluation of resistance prediction in AML patients at<a href="http://www.viviabiotech.com/">Vivia Biotech</a></p>

<p>Data analysis of pharmacodynamics of different anti-cancer agents over clinical samples. DEveloping different approaches and non linear mathematical models to evaluate the resistance of the ex-vivo AML cells to different anti-cancer therapies.</p>

<!--<p><code class="highlighter-rouge">2015-2016</code>
<strong>ML Scientist, <a href="http://www.openbiome.org/">OpenBiome</a> / <a href="http://www.finchtherapeutics.com/">Finch Therapeutics</a></strong> Cambridge,MA (remote)</p>

<ul>
  <li>Statistical analysis of microbiome communities during fecal transplant delivery by capsule</li>
  <li>Project consulting for large pharma organization to analyze prevalence of drug resistant genes in the general population</li>
  <li>Retrospective analysis of Phase 2b trial results and additional microbiome sequencing to measure efficacy</li>
</ul>

<p><code class="highlighter-rouge">2014-2015</code>
<strong>Chief Scientist, Klappo/um.ai,</strong> London,UK</p>

<ul>
  <li>Building a recommendations engine to craft context-aware suggestions for every meal.</li>
  <li>Normalization and concept mapping used to organize natural language data related to food onto a structured ontology</li>
  <li>Data <a href="https://elipapa.github.io/youarewhatyoueat/">analysis and visualization</a></li>
  <li>management team responsible for the business plan and investor relations</li>
  <li>medical and scientific direction</li>
</ul>

<p><code class="highlighter-rouge">2014</code>
<strong>Junior Doctor, <a href="http://www.imperial.nhs.uk/">Imperial College NHS Trust</a></strong>,North West London, UK</p>

<ul>
  <li>General Practice</li>
  <li>Care of Elderly, orthopaedic outreach</li>
  <li>Anaesthetics &amp; ITU</li>
  <li>Gastroenterology &amp; Acute medical take</li>
</ul>

<p><code class="highlighter-rouge">2013</code>
<strong>Theoretical System Biology group, Prof. M. Stumpf</strong>, Imperial College</p>

<ul>
  <li>Integrating Chip-seq, RNAseq and transcriptomics data describing nitrogen stress response in e.coli</li>
  <li>Organized query with a relational db, structured ontology and visualization</li>
  <li>Bayesian model selection of potential e.coli nitrogen stress pathways</li>
</ul>

<p><code class="highlighter-rouge">2012</code>
<strong>Consultant, SERES Health,</strong> Cambridge, MA (remote)</p>

<ul>
  <li>Computationally designed and predicted fitness of synthetic 
microbial communities intended for therapeutic transplantation using 
unsupervised/supervised learning..</li>
  <li>Provided strategic input and scientific advice.</li>
</ul>

<p><code class="highlighter-rouge">2009-2013</code>
<strong>Alm Laboratory for Microbiology, Prof. Eric J. Alm</strong>, MIT</p>

<ul>
  <li><a href="http://nbviewer.jupyter.org/github/elipapa/SLiMEbook/blob/master/SLiMEbook.ipynb">Supervised classification of microbiome samples</a> from pediatric IBD patients</li>
  <li>Feature extraction using ecological and phylogenetic information</li>
  <li>Quality control and downstream analysis of Human Microbiome Project sequencing pipeline</li>
</ul>

<p><code class="highlighter-rouge">2009</code>
<strong>Founder, Enumeral diagnostics,</strong> Cambridge, MA</p>

<ul>
  <li>MIT $100K Entrepeneurship competition semifinalist</li>
  <li>Developed high-throughput data acquisition and analysis platforms, scaling the technology to production levels.</li>
  <li>Contributed to the development of the microfluidic platform at the core of the company intellectual property</li>
</ul>

<p><code class="highlighter-rouge">2006-2009</code>
<strong>Laboratory of Hidde L. Ploegh</strong>, Whitehead Institute, MIT</p>

<ul>
  <li>Unsupervised learning to map affinity and isotype of secreted antibodies in individual primary B cells.</li>
  <li>Describing immune response progression by statistical modelling of the B cell population</li>
  <li>Created a cluster pipeline to automate image analysis of fluorescence microscopy data</li>
  <li>Murine antibody cloning and expression; fluorescence tagging</li>
  <li>Real time fluorescence microcopy; advanced image analysis</li>
</ul>

<p><code class="highlighter-rouge">2004-2005</code>
<strong>Biomedical Nanotechnology Group, Prof. W C. Chan</strong>, University of Toronto</p>

<ul>
  <li>Nanoparticles cytotoxicity</li>
  <li>Quantum Dots characterization (TEM, Absorption, PL, X-IRD)</li>
  <li>Real time fluorescence microscopy, single molecule spectroscopy and biophysics.</li>
</ul>

<p><code class="highlighter-rouge">2003</code>
<strong>Biomaterials Group, Prof. M.C.Tanzi</strong>, Politecnico di Milano, Italy</p>

<ul>
  <li>Morphological, mechanical and functional characterization of polyurethane scaffolds for tissue engineering applications.</li>
</ul>

<h2 id="education">Education</h2>

<p><code class="highlighter-rouge">2013</code>
<strong>MBBS, Imperial College London</strong>
Medicine &amp; Surgery</p>

<p><code class="highlighter-rouge">2006-2012</code>
<strong>Ph.D, Harvard Medical School/Massachusets Institute of Technology</strong></p>

<ul>
  <li>Medical Engineering &amp; Medical Physics, <a href="http://hst.mit.edu/">Harvard/MIT HST Institute</a></li>
  <li>Thesis: <a href="https://speakerdeck.com/u/elipapa/p/phd-defense-talk-1">High-throughput experimental and computational tools for exploring immunity and the microbiome</a></li>
</ul>

<p><code class="highlighter-rouge">2008</code>
<strong>Sc.M., Massachusets Institute of Technology</strong>
Mechanical Engineering</p>

<p><code class="highlighter-rouge">2005</code>
<strong>BASc (Honors), University of Toronto</strong>
Engineering Science, Biomedical Option</p> -->

<h3 id="computing">Computing</h3>

<p>Python - jupyter, scikit-learn, numpy, matplotlib, seaborne, pandas, lxml, NLTK, requests, sqlite3.
R - caret, shiny, ggplot2, knitr, shiny.
Web development - working knowledge of HTML5 (CSS and Javascript ongoing)
Linux - Git, bash
Neo4J, Oracle ,PostgreeSQL </p>

<p>Sketchnotes, and scientific illustrations and animations for papers or tutorial purposes.</p>

<!--<p>Contributor to open source projects: <a href="https://github.com/opentargets">open targets</a> , <a href="https://github.com/kubernetes/kubernetes">kubernetes</a>, <a href="https://github.com/spotify/luigi">luigi</a></p>

<h2 id="fellowships--awards">Fellowships &amp; Awards</h2>

<p><code class="highlighter-rouge">2012</code>
Bursary recipient, Exploring Human Host-Microbiome Interactions in Health and Disease, Wellcome Trust Scientific Conferences</p>

<p><code class="highlighter-rouge">2010-2011</code>
Postgraduate D Scholarship, National Science Engineering Research Council, Canada</p>

<p><code class="highlighter-rouge">2008-2009</code>
Poitras pre-doctoral fellowship</p>

<p><code class="highlighter-rouge">2007</code>
Martino Scholar, Harvard/MIT Health Science Tech. Inst.</p>

<p><code class="highlighter-rouge">2008</code>
Martha Gray Prizes for Excellence in Research, Annual Forum, Harvard/MIT Health Science Tech. Inst.</p>

<p><code class="highlighter-rouge">2008</code>
Competition Semifinalist, MIT 100k Business Plan</p>

<p><code class="highlighter-rouge">2005-2008</code>
Postgraduate M Scholarship, National Science Engineering Research Council, Canada</p>

<p><code class="highlighter-rouge">2004</code>
University of Toronto Life Sciences Award, University of Toronto</p>

<p><code class="highlighter-rouge">2004</code>
NSERC Summer Research Award, National Science Engineering Research Council, Canada</p>

<p><code class="highlighter-rouge">2003</code>
OUA Academic Achievement Award, Ontario, Canada</p>

<p><code class="highlighter-rouge">2003</code>
#2 Canadian Army University Course Undergrad Scholarship,University of Toronto</p>

<p><code class="highlighter-rouge">2002-04</code>
Silver T – academic athletic excellence, University of Toronto</p>

<p><code class="highlighter-rouge">2001</code>
Ontario Scholar, Government of Ontario, Canada</p>

<h2 id="publications">Publications</h2>-->

<!-- A list is also available [online](http://scholar.google.co.uk/citations?user=LTOTl0YAAAAJ) -->

<!--
<h3 id="journals">Journals</h3>

<p><code class="highlighter-rouge">2017</code>
Koscielny G, An P, Carvalho-Silva D, Cham JA, Fumis L, Gasparyan R, Hasan S, Karamanis N, Maguire M, <em>Papa E</em>, Pierleoni A, Pignatelli M, Platt T […] Birney E, Barrett J, Dunham I.
<a href="https://www.ncbi.nlm.nih.gov/pubmed/27899665">Open Targets: a platform for therapeutic target identification and validation.</a> <strong>Nucleic Acids Res</strong>. 2017 Jan 4;45(D1):D985-D994.
doi: 10.1093/nar/gkw1055. PubMed PMID: 27899665; PubMed Central PMCID:
PMC5210543.</p>

<p><code class="highlighter-rouge">2017</code>
Fischer M, Bittar M, <em>Papa E</em>, Kassam Z, Smith M.
<a href="http://www.tandfonline.com/doi/abs/10.1080/19490976.2017.1283469?journalCode=kgmi20">Can You Cause Inflammatory Bowel Disease with Fecal Transplantation?</a>
 <strong>Gut Microbes</strong> 2017 Jan 19:0. doi: 10.1080/19490976.2017.1283469</p>

<p><code class="highlighter-rouge">2012</code>
<em>Corina Duenas</em>, Michael Docktor, Christopher Smillie, Sarah Weber,
 Sarah Pacocha Preheim,Dirk Gevers,Georgia Giannoukos, Dawn Ciulla, 
Diana Tabbaa, Jay Ingram, David B Schauer, Doyle V Ward, Joshua R 
Korzenik, Ramnik J Xavier, Athos Bousvaros, Eric J Alm.
<a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0039242">Non-invasive mapping of the gastrointestinal microbiota identifies children with inflammatory bowel disease.</a>
<strong>PLoS ONE</strong> 2012;7(6):e39242.</p>

<p><code class="highlighter-rouge">2011</code>
Rhiannon White, Sachiko Miyata, <em>Corina Duenas</em>, Eric Spooner, Kleoniki Gounaris, Murray Selkirk, Katerina Artavanis-Tsakonas.
<a href="http://www.plosntds.org/article/info%3Adoi%2F10.1371%2Fjournal.pntd.0001340">Characterisation
 of the Trichinella spiralis deubiquitinating enzyme, TsUCH37, an 
evolutionarily conserved proteasome interaction partner.</a>
<strong>PLoS Negl Trop Dis.</strong> 2011 Oct;5(10):e1340.</p>

<p><code class="highlighter-rouge">2011</code>
Katerina Artavanis-Tsakonas, Pia V Kasperkovitz, <em>Corina Duenas</em>, Michael L Cardenas, Nida S Khan, Annemarthe G Van der Veen, Hidde L Ploegh and Jatin M Vyas.
<a href="http://iai.asm.org/content/79/3/1098.long">The Tetraspanin CD82 is Specifically Recruited to Fungal and Bacterial Phagosomes Prior to Acidification.</a>
<strong>Infection and Immunity</strong> 2011 79(3):1098-106\</p>

<p><code class="highlighter-rouge">2009</code>
Adebola Ogunniyi, Craig Story, <em>Corina Duenas</em>, Eduardo Guillen, J. Christopher Love.
<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4034573/">Screening Individual Hybridomas by Microengraving to Discover Monoclonal Antibodies.</a>
<strong>Nature Protocols</strong> 2009 4(5):767-82</p>

<p><code class="highlighter-rouge">2009</code>
Jehnna L. Ronan, Craig Story, <em>Corina Duenas</em>, J. Christopher Love.
<a href="http://www.sciencedirect.com/science/article/pii/S0022175908003268">Optimization of the surfaces used to capture antibodies from single hybridomas reduces the time required for microengraving.</a>
<strong>Journal of Immunological Methods</strong> 2009, 340(2):164-9\</p>

<p><code class="highlighter-rouge">2008</code>
Craig Story*, <em>Corina Duenas* (co-author)</em>, Chih-Chi Andrew Hu, Jehnna L Ronan, Hidde L Ploegh, J.Christopher Love.
<a href="http://www.pnas.org/content/105/46/17902.full">Profiling Antibody Responses by Multiparametric Analysis of Single B Cells.</a>
<strong>PNAS</strong> 2008 105(46):17902-7</p>

<p><code class="highlighter-rouge">2005</code>
Hans Fischer, <em>Eli Papa</em>, Lichuan Liu, K. Sandy Pang, Warren C. W. Chan.
<a href="http://proceedings.spiedigitallibrary.org/proceeding.aspx?articleid=877794">Preliminary Results: Exploring the Interactions of Quantum Dots with Whole Blood Components.</a>
<strong>SPIE Proceedings</strong> 2005 5969,54</p>

<p><code class="highlighter-rouge">2004</code>
Wen Jiang, <em>Eli Papa</em>, Hans Fischer, Sawitri Mardyani, Warren C.W. Chan.
<a href="http://www.cell.com/trends/biotechnology/abstract/S0167-7799(04)00299-9?_returnURL=http%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0167779904002999%3Fshowall%3Dtrue">Semiconductor quantum dots as contrast agents for whole animal imaging.</a>
<strong>Trends in Biotechnology</strong> 2004 22:12</p>

<h3 id="posters">Posters</h3>

<p><code class="highlighter-rouge">2016</code>
Corina Duenas, Andrea Pierleoni, Gautier Koscielny, Naruemon Pratanwanich,
 Oliver Stegle, Jeffrey Barrett, Ian Dunham, and the Open Targets Team
Open Targets : a platform for therapeutic target identification and 
validation
<em>MASAMB 2016</em></p>

<p><code class="highlighter-rouge">2016</code>
Miguel Pignatelli, Denise Carvalho-Silva, Jennifer Cham, Luca Fumis, 
Samiul Hasan,
Nikiforos Karamanis, Gautier Koscielny , Michael Maguire, William 
Newell, Corina Duenas, Andrea Pierleoni, Francis Rowland, Sirarat 
Sarntivijal, Oliver Stegle, Jessica Vamathevan, Ewan Birney, Jeffrey 
Barrett, Ian Dunham and the Open Targets team
Target Validation Platform: A New Drug Target Identification and 
Prioritisation Web Platform
<em>Genome informatics 2016</em></p>

<p><code class="highlighter-rouge">2016</code>
Monika Fischer, Mohamad Bittar, Corina Duenas, Zain Kassam, Mark Smith
Can you cause IBD with fecal transplantation?: 31-patient case series of
 fecal transplantation using stool from a donor who later developed 
Crohn’s disease
<em>Digestive Disease Week 2016</em></p>

<p><code class="highlighter-rouge">2016</code>
Jessica Allegretti, Monika Fischer, Corina Duenas, Ryan Elliot, M. Jane Klank, Gina Mendolia, Elaine Vo, Zain Kassam, Mark Smith
Fecal Microbiota Transplantation Delivered via Oral Capsules Achieves Microbial Engraftment Similar to Traditional
Delivery Modalities: Safety, Efficacy and Engraftment Results From a Multi-Center Cluster Randomized Dose-Finding Study
<em>Digestive Disease Week 2016</em></p>

<p><code class="highlighter-rouge">2012</code>
White RR, Morrow M, Miyata S, Papa E, Spooner E, Selkirk M, Gounaris K, Das C, Artavanis-Tsakonas K
Characterisation of the Trichinella Spiralis Deubiquitinating Enzyme, TsUCH37
<em>Molecular and Cellular Biology of Helminth Parasites VII</em></p>

<p><code class="highlighter-rouge">2012</code>
Corina Duenas, Michael Docktor, Christopher Smillie, Sarah Weber, Sarah P.
 Preheim, Dirk Gevers, Georgia Giannoukos, Dawn Ciulla, Diana Tabbaa, 
Jay Ingram, David B Schauer, Doyle V Ward, Joshua R Korzenik, Ramnik J 
Xavier, Athos Bousvaros, Eric J Alm.
<a href="http://eliseopapa.org/downloads/documents/ibdML-poster.pdf">Diagnosing IBD from the fecal microbiome</a>
<em>Exploring Human Host-Microbiome Interactions in Health and Disease, Wellcome Trust Scientific Conferences</em></p>

<p><code class="highlighter-rouge">2008</code>
High-Throughput and High-Content Screening of Antibody Responses from Single Cells
<em>AIChE annual meeting, Nanoscale Science Engineering Forum</em></p>

<p><code class="highlighter-rouge">2008</code>
Applying Ligands to B Cell Receptors by Microfluidics
<em>AIChE annual meeting, Engineering Fundamentals in Life Sciences</em></p>

<p><code class="highlighter-rouge">2008</code>
Microengraving for high-throughput affinity mapping of humoral responses
<em>Harvard/MIT HST Forum</em></p>

<p><code class="highlighter-rouge">2008</code>
Multi-variate profiling of B cell immune responses
<em>Novartis Vaccine Symposium</em></p>

<h3 id="patents">Patents</h3>

<p><code class="highlighter-rouge">2009</code>
Composition of an Array of Microwells with an Integrated Microfluidic System, <a href="https://patents.google.com/patent/US8569046B2/en?inventor=eliseo+papa&amp;oq=eliseo+papa">US8569046B2</a></p>

<h2 id="volunteering">Volunteering</h2>

<p><code class="highlighter-rouge">2005</code>
<strong>Field Operative, AISPO, San Raffaele del Monte Tabor Foundation.</strong> Milan, Italy</p>

<ul>
  <li>Streamlined diagnostic routines, Kampala’s Hospital &amp; Gulu’s outpost, Uganda</li>
  <li>Consulted regarding the infrastructure, human resources and logistics of the Kampala’s hospital</li>
</ul>

<p><code class="highlighter-rouge">2004</code>
<strong>Engineers Without Borders.</strong></p>

<ul>
  <li>University of Toronto Conference delegate</li>
</ul>

    </div>-->
  </div>



</body></html>